src/marketplace/sellers/malware-scan.md

```diff
@@ -0,0 +1,46 @@
+---
+group: marketplace-sellers
+title: Malware Scan
+---
+
+## Overview
+
+Malware Scan checks that provided extension and all media files and documents do not contain any malicious code or links.
+
+## Why we test?
+
+Security is one of the top concerns for Magento. And we would like to provide same level of certainty for 3rd party extensions distributed at [Magento Marketplace](https://marketplace.magento.com/) as we have for Magento core components.
+
+## When we test?
+
+All uploaded files including package with submitted extension and all associated media files and documents are scanned before all other checks.
+
+If Malware Scan is not passed submission will not be taken for further verification and validation.
+
+## What we are looking for?
+
+We check all files for:
+
+1. Signatures of known viruses and malware software.
+1. Links to sites with with harassment content.
+
+## What tools and environment we are using?
+
+To perform Malware Scan we use several tools:
+
+1. General purpose antivirus with automatically updated virus database.
+1. [Yara](https://github.com/virustotal/yara) with set of Magento-specific rules.
+
+## How read an error report?
+
+Magento Developer Portal will notify user about detected malware software or malicious link during files uploading.
+
+If you receive such method please check integrity of submitted extension source code and scan environment where package was generated with an antivirus.
```

Comment:
Please rewrite this sentence as it does not make very good grammatical sense in English